devices/xiaomi.js

```diff
@@ -1381,6 +1381,26 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.curtain.acn003'],
+        model: 'ZNCLBL01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara curtain driver E1',
+        fromZigbee: [fz.battery, fz.xiaomi_curtain_position_tilt, fz.aqara_opple],
+        toZigbee: [tz.xiaomi_curtain_position_state, tz.ZNCLBL01LM],
+        exposes: [e.cover_position().setAccess('state', ea.ALL), e.battery(),
+            e.battery_voltage().withAccess(ea.STATE_GET),
+            exposes.enum('motor_state', ea.STATE, ['stopped', 'opening', 'closing'])
+                .withDescription('Motor state'),
+            exposes.binary('running', ea.STATE, true, false)
+                .withDescription('Whether the motor is moving or not'),
+            exposes.enum('hooks_state', ea.STATE_GET, ['Loosened', 'Tightened', 'Tightening', 'Loosening'])],
```

Comment:
I also think `hooks_state` should be brought to a single style:

```javascript
            exposes.enum('hooks_state', ea.STATE_GET, ['loosened', 'tightened', 'tightening', 'loosening'])
                .withDescription('Hooks state')],
```

***

Are you sure that `[aqaraOpple] 1064 = 0` is `loosened` and not `stopped`?

It's not clear from the description:
> press button 2 times again to stop loosening the hooks

Still, the loosening can be stopped in an intermediate state, or the hooks are only loosened completely, and what value is reported in these cases: the same or different?
And the same question, but already when tightening.

Reply:
1064 =0 when either the hook is completely loosened or stopped in the middle.   But when the hook is tightened, 1064 returns 1.  So the description for 1064=0 can be either way.  :)  
Or, we can use the description which I found might be a similar model in China. 
https://home.miot-spec.com/spec/lumi.curtain.hmcn02

In PIID 10 (lock_unlock_status), they use these description instead:
0 - Unlocked
1 - Locked
2 - Lock-seting
3 - Unlock-seting

What do you think?

Reply:
Ok, now it's clear 🙂

I think we will use the state names from a similar model:

```javascript
            exposes.enum('hooks_state', ea.STATE_GET, ['unlocked', 'locked', 'locking', 'unlocking'])
                .withDescription('Hooks state')],
```

***

Also we need to add to `aqara_opple`:

```javascript
        case '1064':
            if (['ZNCLBL01LM'].includes(model.model)) {
                payload.hooks_state = {0: 'unlocked', 1: 'locked', 2: 'locking', 3: 'unlocking'}[value];
            }
            break;
```